packages/client/components/GitLabScopingSearchFilterMenu.tsx

```diff
@@ -142,6 +144,11 @@ const GitLabScopingSearchFilterMenu = (props: Props) => {
               : [...selectedProjectsIds, projectId]
             gitlabSearchQuery.setValue(newSelectedProjectsIds, 'selectedProjectsIds')
           })
+          SendClientSegmentEventMutation(atmosphere, 'Selected GitLab Project Filter', {
```

Comment:
-1 I think for future analysis it would be better if we keep these events more generic `'Selected Poker Scope Project Filter'` and adding `service: 'gitlab'` as a property.